server/lib/nl/constants.py

```diff
@@ -179,6 +179,8 @@
     'sf north bay',
     'sf south bay',
     'sf east bay',
+    # US
+    'united states',  # need this because the word "states" gets replaced.
```

Comment:
Just to confirm, this is not due to AC change, but something related to NER being made here?

Reply:
Yes, correct! This was needed not due to NER being flaky but because of the stop words removal removing "states" :) Nothing comes for free :(